src/main/scala/beam/physsim/jdeqsim/JDEQSimRunner.scala

```diff
@@ -330,9 +331,11 @@ object JDEQSimRunner {
                   caccSettings.roadCapacityAdjustmentFunction.getCapacityWithCACCPerSecond(link, caccShare, time)
                 //volume is calculated as number of vehicles entered the road per hour
                 //capacity from roadCapacityAdjustmentFunction is number of vehicles per second
-                val tmp = volume / (capacity * 3600)
-                val result = ftt * (1 + tmp * tmp)
 
+                val alpha = link.getAttributes.getAttribute("alpha").toString.toDouble
+                val beta = link.getAttributes.getAttribute("beta").toString.toDouble
```

Comment:
I think we could use `.asInstanceOf[Double]` instead of formatting a Double and parsing it back.

Reply:
Good catch, adjusted